public/manifest.json

```diff
@@ -20,7 +20,7 @@
     }
   ],
   "start_url": "./",
-  "display": "standalone",
```

Comment:
display Property changed to `fullscreen` 

> Because it Opens the web application without any browser UI and takes up the entirety of the available display area

Reply:
From [MDN regarding `standalone`](https://developer.mozilla.org/en-US/docs/Web/Manifest/display): 

> In this mode, the user agent will exclude UI elements for controlling navigation, but can include other UI elements such as a status bar.

This seems like UI that's worth keeping. Fullscreen views make sense for media consumption apps, but I'm not sure about a communication app. Toggling a fullscreen view for video calls makes sense, but it might be jarring for users if the entire experience is in fullscreen from the start.

FWIW, I can't actually see a see a difference in the browser UI when I tested this change in Chrome. What differences are you seeing in your testing?

Reply:
It looks like you responded in [this thread](https://github.com/jeremyckahn/chitchatter/pull/80/files#r1045238506) but I'll respond here.

This seems to have a more significant effect on Android devices. I use an iPhone so I didn't see this before.

Good thinking! :+1: